.homeycompose/capabilities/measure_runmode.json

```diff
@@ -4,8 +4,9 @@
     "en": "Is ready",
```

Comment:
Is ready is quite confusing, as the value doesn't tell that the spa is ready for anything.
Maybe rename this to "Run mode", which just returns the same as heater_mode.

Reply:
Ah but you are deprecating it. But for the time being, it could be retitled.